services/alertmanager/alertmanager.go

```diff
@@ -130,6 +131,33 @@ receivers:
 		err = ioutil.WriteFile(alertmanagerBaseConfigPath, []byte(defaultBase), 0644) //nolint:gosec
 		svc.l.Infof("%s created: %v.", alertmanagerBaseConfigPath, err)
 	}
+
+	svc.l.Infof("checking %s content", alertmanagerConfigPath)
+	content, err := ioutil.ReadFile(alertmanagerConfigPath)
+	if err != nil {
+		svc.l.Errorf("Failed to load alertmanager config %s: %s", alertmanagerConfigPath, err)
+	}
+
+	// copy the base config if `/etc/alertmanager.yml` is already present but does not have any config.
+	if string(content) == "---\n" {
```

Comment:
added this check because `/etc/alertmanager.yml` is already present having `---\n` as its content when we do `make env-up`